modules/integration-vectors/src/index.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import { NotSupported } from '@aws-crypto/material-management'
+
 export * from './types'
 export * from './get_decrypt_test_iterator'
 export * from './build_get_keyring'
+export { NotSupported }
```

Comment:
you can just `export { NotSupported } from '@aws-crypto/material-management'`
But why?

Reply:
Wait.... I do not know what happened here... I think this is from an earlier effort where I did not move NotSupported... I will remove this.
NotSupported should be tied to Needs every where it is... needed.